front-end-notes/psuedo-selectors.md

```diff
@@ -0,0 +1,24 @@
+#Psuedo-class selectors
+
+###A CSS pseudo-class is a keyword added to selectors that specifices a speical state of the element to be selected.
+
+##### Example
+```css
+ selector: pseduo-class {
+  property: value;
+  };
+```
+Example 2
+```css
+img: hover {
+ color: #000000;
+}
+```
+
+##### References
+
+<ul>
+<li>https://github.com/nashville-software-school/front-end-curriculum/tree/master/11-css-102</li>
+<li>https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/Selectors

```

Comment:
Your missing a closing <li> tag. Also, markdown has it's own syntax for lists. Just use a *